docs/nix.rst

```diff
@@ -17,6 +17,25 @@ Please follow the installation instructions in the `nix manual`_ and additional
     nix-channel --update
     export NIXPKGS_ALLOW_UNFREE=1
 
+Binary Caching
+--------------
+
+Binary caching of Nix dependencies heavily speeds up build times by getting the heavy dependencies (like Psi4) prebuilt from the cache (usually hours). A binary cache for the open source components for a common workstation configuration is provided:
+
+- SSE exentensions up to SSE4.2
+- AVX extensions up to AVX2
+- BLAS and LAPACK is provided by OpenBLAS
+- MVAPICH2 is used as the commond MPI provider
+- The network for MPI parallelisation is ethernet and the loopback interface
+
+The caching is provided by the Cachix_ service and needs to be enabled for your nix installation:
+
+.. code-block: bash
+    nix-env -iA nixos.cachix
+    cachix use chemix
+
+Cachix will (necessarily) add a new :code:`substituters` and :code:`trusted-public-keys` to your :code:`$HOME/.config/nix/nix.conf` if executed as normal user or :code:`/etc/nix/nix.conf` if executed as root. If written to your user configuration, you need to be a configured as :code:`trusted-users` for nix in :code:`/etc/nix/nix.conf` or your new caching settings will be ignored.
```

Comment:
you need to be [a] configured as ... superfluous "a"

Reply:
Changed and added more fitting wording for configuration according to the new `config.nix` in  970cf2d.